src/libraries/System.Security.Cryptography.Pkcs/tests/Pkcs12/ShroudedKeyBagTests.cs

```diff
@@ -40,7 +40,7 @@ public static void BuildWithCharsFactoryReadDirect()
                     Assert.True(rsa2.TrySignData(
                         keyBag.EncryptedPkcs8PrivateKey.Span,
                         sig,
-                        HashAlgorithmName.MD5,
+                        HashAlgorithmName.SHA256,
```

Comment:
The tests in this file did not seem like they were explicitly testing MD5 for any particular reason, so I moved them to SHA256. If this is incorrect, I can change it back and put a condition on the test instead.

Reply:
There is/was probably an element of intermixing all the hash algorithms to just generally make sure they all work as expected.

Since this isn't changing the data we are working with in the PFX, just "do the keys work?", it seems like a fine change.